firmware/applications/SignBuddy/main.c

```diff
@@ -109,22 +109,38 @@ int main(void)
 
   flex_task_setup();
   tsc_task_setup();
+  imu_task_setup();
   sensors_task_setup();
 
   LOG_INFO("App started\r\n");
 
-  RTOS_ERR_CHECK(xTaskCreate(leds_task,
-                             "led_task",
```

Comment:
why did you remove this?

Reply:
I didn't really delete it but it's commented out because it was causing an issue. For some reason, LED task is not letting the program to run.